teams/vsp/teams/product-dev-support/guidelines-for-reviewing-code.md

```diff
@@ -0,0 +1,81 @@
+# Guidelines for reviewing code
+
+## Background
+
+This doc provides guidelines for VSP engineers doing code reviews for VFS teams: how they should do them, and what emphasis we want to put in those guidelines.
+
+The [guidelines for VFS teams](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/engineering/code_review_guidelines.md) are public; this doc is a VSP-team corollary to that one.
+
+
+## Frontend review guidelines (vets-website)
+
+These guidelines are for VSP engineers reviewing code in the `vets-website` repository.
+
+### Responsibilities 
+
+*   overall stability of `vets-website`
+*   site performance 
+*   enforcing test coverage 
+*   enforcing linting rules
+
+### Examples of responsibilities 
+
+*   ensuring that external teams are using the VA Design System when possible (i.e. not creating unnecessary CSS)
+*   ensuring changes to common helpers in the `platform/` and `site/` won't impact other apps teams 
+*   `/platform/site-wide` and `**/*.drupal.liquid` templates are in the shared folders but are generally maintained by the Global UX / CMS teams 
+
+### Examples of things that should block approval
+
+*   The pull request is too large (i.e. more than 300 lines)
+*   The changes excessively circumvent rules enforced by CI (e.g. disabling linting, writing unit tests that obviously don’t test anything) 
+    *   _Note: there are legitimate reasons to disable linting rules, but use your best judgement. Generally if it’s not a rule that’s been skipped somewhere else in the repo, it’s probably not something we should approve. _
+*   The changes impact shared components / helpers but the PR does not document adequate regression testing was performed
+    *   Changes to /platform or /site folders (especially to the forms system) should be made by or closely supervised by the Tools and Improvements team 
+*   Looping / control structures that obviously could result in unresponsive UIs (e.g. using a `do-while` loop that mutates its terminating condition)
+*   Adding excessive or large static assets to the repo
+    *   We don’t have any image optimization / compression in the build process
+*   Adding files that should be somewhere else (e.g. HTML files should be in the Drupal CMS or the `vagov-content` repo) 
+
+### Examples of things that should not block approval
+
+These are things you can optionally look at, but aren't required to enforce outside of `platform/` and `site/`. We generally offer recommendations, but don't block merging, for these kinds of issues:
+
+*   Poor code style / non-idiomatic JS usage 
+    *   _i.e. using a `for` loop instead of a `.map` _
+    *   _i.e. using conditionals in a loop instead of `.filter` _
+*   Inconsistency with our own style (for other repositories)
+    *   _I.e. ensuring the general approach to GIBCT is maintained is okay. Even if that style doesn’t match our other repositories, it is internally consistent _
```

Comment:
@rianfowler i think you had a bad edit in this line? reads a little weird to me (... is maintained is okay)